### DISCUSSION TITLE
Control parameters in masks

### DISCUSSION THREAD

**Discussion by 3songs** [rsjaffe/MIDI2LR#741] [Q&A]
I switched from Loupedeck to Midi2LR (with Behringer X Touch) just recently. 
I wonder if I missed out on something: I sadly miss the possibility to change parameters IN MASKS. E.g. if I select 'mask subject', how do I get the rotary control to work on the exposure setting of the mask and not the overall exposure of the whole photo?

**Comment by antzpantz** [comment]
Given rsjaffe hasn't been actively developing the plug-in for nearly a year
(I hope you're ok, mate), I can only assume that Lightroom has changed
substantially in that time that the MIDI2LR functionality hasn't been
implemented or the API hasn't allowed that yet.

**Reply by 3songs** [reply]
I think it should be possible because Loupedeck can do it. The point is: the controls should affect the layer/mask, that has the focus. Right now my controls seem like 'hardwired' to the 'main layer', they affect the whole image and not the selections (e.g. subject, sky, background). I hope the development of Midi2Lr goes on... because 'yes', Lightroom has changed a lot recently. Many possibilities that were once reserved for Photoshop are now (to a lesser degree) implemented in LR.

**Reply by remov-b4-flight** [reply]
Isn't it about 'local Adjustment' ?

**Reply by 3songs** [reply]
Yes, that's what I'm looking for: the controls should affect or local or global. Local is chosen in LR by Shift+W (Windows) - Masking.
